patch.yml

```diff
@@ -13,6 +13,54 @@
 # limitations under the License.
 
 ---
+- name: Play | Set correct RDBMS release and set fact if this is a single node inventory
+  hosts: dbasm
+  gather_facts: false
+  tasks:
+    - name: include defaults
```

Comment:
It is posisble to prefix task names with the playbook name and a vertical bar like (most of) the rest of the toolkit?

Reply:
Ack, will work on it

Reply:
Done